frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -2,6 +2,10 @@ import * as React from 'react';
 
 import { resourceSidebars } from './resource-sidebars';
 
+export const sidebarScrollTop = () => {
```

Comment:
Not necessarily something that needs to be fixed now, but this could probably be implemented as a reusable hook. Something similar to the `useRefWidth` hook.